docs/data/material/getting-started/templates/checkout/getCheckoutTheme.tsx

```diff
@@ -0,0 +1,651 @@
+                  backgroundColor: alpha(brand[600], 0.3),
+                  borderColor: brand[700],
+                },
+              }),
+              ...(ownerState.variant === 'text' && {
+                color: brand[300],
+                '&:hover': {
+                  backgroundColor: alpha(brand[600], 0.3),
+                  borderColor: brand[700],
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiCard: {
+        styleOverrides: {
+          root: ({ theme, ownerState }) => ({
+            backgroundColor: gray[50],
+            borderRadius: 10,
+            outline: `1px solid ${alpha(gray[200], 0.8)}`,
+            boxShadow: 'none',
+            transition: 'background-color, border, 80ms ease',
+            ...(ownerState.variant === 'outlined' && {
+              boxSizing: 'border-box',
+              background: `linear-gradient(to bottom, #FFF, ${gray[50]})`,
+              '&:hover': {
+                borderColor: brand[300],
+                boxShadow: `0 0 24px ${brand[100]}`,
+              },
+            }),
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: alpha(gray[800], 0.6),
+              outline: `1px solid ${alpha(gray[700], 0.3)}`,
+              ...(ownerState.variant === 'outlined' && {
+                boxSizing: 'border-box',
+                background: `linear-gradient(to bottom, ${gray[900]}, ${alpha(
+                  gray[800],
+                  0.5,
+                )})`,
+                '&:hover': {
+                  borderColor: brand[700],
+                  boxShadow: `0 0 24px ${brand[800]}`,
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiDivider: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            borderColor: `${alpha(gray[200], 0.8)}`,
+            ...(theme.palette.mode === 'dark' && {
+              borderColor: `${alpha(gray[700], 0.4)}`,
+            }),
+          }),
+        },
+      },
+      MuiPaper: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            backgroundImage: 'none',
+            backgroundColor: gray[100],
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: gray[800],
+            }),
+          }),
+        },
+      },
+      MuiOutlinedInput: {
+        styleOverrides: {
+          notchedOutline: {
+            border: 'none',
+          },
+          root: ({ theme }) => ({
+            '& .MuiInputBase-input': {
+              '&::placeholder': {
+                opacity: 0.7,
+                color: gray[500],
+              },
+            },
+            boxSizing: 'border-box',
+            flexGrow: 1,
+            maxHeight: 40,
+            height: '100%',
+            borderRadius: '10px',
+            border: '1px solid',
+            borderColor: gray[200],
+            boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.1)',
```

Comment:
```suggestion
            boxShadow: '0px 1px 1px rgba(0, 0, 0, 0.1)',
```

Removing the blur makes it look a bit tighter